src/OmniSharp.Host/MSBuild/Discovery/Providers/VisualStudioInstanceProvider.cs

```diff
@@ -17,7 +17,7 @@ public VisualStudioInstanceProvider(ILoggerFactory loggerFactory)
 
         public override ImmutableArray<MSBuildInstance> GetInstances()
         {
-            if (!PlatformHelper.IsWindows)
+            if (!PlatformHelper.IsWindows || PlatformHelper.IsMono)
```

Comment:
Could you remove the first condition?

```C#
if (PlatformHelper.IsMono)
{
    return NoInstances;
}
```

Reply:
Sure,  but I figured that was also there for non-windows non-mono platforms e.g. netcore?

Reply:
OmniSharp does not run on .NET Core. It requires Mono or Desktop CLR. So, that change should be fine.

Reply:
Cool.  Done.